coordinator/src/node.rs

```diff
@@ -164,26 +169,25 @@ impl Node {
         let accept_settlement_amount = calculate_accept_settlement_amount(
             opening_price,
             closing_price,
-            trade_params.quantity,
+            position.quantity,
             leverage_long,
             leverage_short,
             position.direction,
         )?;
 
         tracing::debug!(
             "Settling position of {accept_settlement_amount} with {}",
-            trade_params.pubkey
+            position.trader.to_string()
```

Comment:
Would be cool if we make it a habbit to move variables out of the message. This will allow us to filter the logs more easily.